camera/rpi-files/setup.cfg

```diff
@@ -3,9 +3,11 @@ multi_line_output=3
 include_trailing_comma=True
 force_grid_wrap=0
 use_parentheses=True
-line_length=88
+line_length=79
```

Comment:
what's this "magic number"

Reply:
79 is standard per PEP8 guidelines. I copied 88 from some article.